### PR TITLE
Milestone 3: Create contact form for My-portfolio project

### PR DESCRIPTION
**Pull request by pasytchangwa**
## Create footer section for My-portfolio project

The following was done:

- Created the contact form 
- Added styling for the contact form
- Verify linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -278,6 +278,29 @@ <h3 class="skill-title">
...
+              <input type="text" name="username" placeholder="Yeremias" value="Yeremias" maxlength="30" required>
+            </li>
+            <li class="second">
+              <input type="email" name="email" placeholder="natajayanj14@gmail.com" value="natajayanj14@gmail.com" required>
+            </li>
+            <li class="third">
+              <textarea name="comment" cols="30" rows="10" placeholder="Write your message here" maxlength="500" required></textarea>
+            </li>
+          </ul>
+          <a href="#" class="btn4">Get in touch</a>
```
Comment (Davidosky007) [inline index.html]: I suggest you wrap the C.T.A with a `button` tag or an input element with type submit for this is good practice, Read [This](https://www.w3schools.com/tags/att_button_form.asp) for more information on how form buttons are created

File: index.html [hunk 2]
```diff
@@ -278,6 +278,29 @@ <h3 class="skill-title">
...
+        </form>
+        <div class="footer-line"></div>
+      </footer>
```
Comment (Davidosky007) [inline index.html]: I think flexbox should be used In order to layout the elements on the page. as stated in the project requirement

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Tchilo @pasytchangwa, 

Good job so far! 💪 

There are some issues that you still need to work on to go to the next project but you are almost there!

CR3_gif_almost_there
 ## Required Changes ♻️
* I left you some comments below, please make the necessary changes 🙏 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

# [OPTIONAL]
* This is an OPTIONAL suggestion but i strongly suggest you make the necessary changes 
* the padding(left, right,) one the class(card) and class(card-section) is making the section overflow kindly replace with just padding.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (pasytchangwa) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (pasytchangwa) [reply index.html]: Done

Comment (Dipeshtwis) [reply index.html]: This was successfully addressed.

File: index.html (hunk 1 above)
Comment (Dipeshtwis) [reply index.html]: This was successfully addressed.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @pasytchangwa, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**